Best practices/Security guidelines.md

```diff
@@ -1,28 +1,19 @@
-In order to pass some of the automated security tests, as well as to keep your .NET application safe it is advised to use some standard security measurements.  This section of the handbook will provide insights into some of the most commonly used strategies.
+During the development of an application, besides worrying about writing clean code with good performance, we must always keep an eye on security. After all, the fastest and prettiest code is not that useful it allows malicious actors to access our users' data. This section of the handbook will provide insights into some of the most commonly used strategies.
```

Comment:
`After all, the fastest and prettiest code is not that useful it allows malicious actors to access our users' data.` I believe it should say `if` instead of `it`